ui/App/SharedComponents/ProjectAnchorHovercard.svelte

```diff
@@ -0,0 +1,132 @@
+<!--
+ Copyright © 2021 The Radicle Upstream Contributors
+
+ This file is part of radicle-upstream, distributed under the GPLv3
+ with Radicle Linking Exception. For full terms see the included
+ LICENSE file.
+-->
+<script lang="ts">
+  import type * as project from "ui/src/project";
+
+  import * as org from "ui/src/org";
+  import * as router from "ui/src/router";
+
+  import {
+    CopyableIdentifier,
+    Hovercard,
+    Icon,
+    IdentifierLink,
+  } from "ui/DesignSystem";
+
+  import OrgIdentity from "ui/App/SharedComponents/OrgIdentity.svelte";
+
+  export let anchor: project.Anchor;
+  export let replicated: boolean = false;
+
+  const openCommit = () => {
+    router.push({
+      type: "project",
+      params: {
+        activeView: { type: "commit", commitHash: anchor.commitHash },
+        urn: anchor.projectId,
+      },
+    });
+  };
+
+  $: anchorColor =
+    anchor.type === "confirmed"
+      ? "--color-primary"
+      : "--color-foreground-level-5";
+  $: title = anchor.type === "confirmed" ? "Anchored in" : "Pending in";
+  $: titleColor =
+    anchor.type === "confirmed"
+      ? "--color-primary"
+      : "--color-foreground-level-5";
+</script>
+
+<style>
+  .header {
+    border-bottom: 1px solid var(--color-foreground-level-2);
+    height: 3.5rem;
+    align-items: center;
+    display: flex;
+    color: var(--color-primary);
+    padding-left: 1rem;
+    padding-right: 1rem;
+  }
+  .meta {
+    display: flex;
+    color: var(--color-foreground-level-6);
+    padding: 1rem 1rem 0 1rem;
+    align-items: center;
+  }
+  .meta:last-of-type {
```

Comment:
Unless there’s a good reason we should use `:first-child` instead of `:first-of-type`. The former describes the purpose more directly and is more robust to changes of element tags.

Reply:
I assume you meant `:last-child`?
8012b6d5b68e0797772b8795809655d098bae091

Reply:
> I assume you meant `:last-child`?

Yes, of course